src/gui/stac/qgsstacdownloadassetsdialog.cpp

```diff
@@ -86,6 +95,25 @@ QStringList QgsStacDownloadAssetsDialog::selectedUrls()
   return urls;
 }
 
+void QgsStacDownloadAssetsDialog::showContextMenu( QPoint p )
+{
+  QTreeWidgetItem *item = mTreeWidget->itemAt( p );
+  if ( !item )
+    return;
+
+  mTreeWidget->setCurrentItem( item );
+  mContextMenu->clear();
+
+  const QString url = item->text( 5 );
+  QAction *copyUrlAction = new QAction( tr( "Copy URL" ), this );
```

Comment:
It probably doesn't matter much, but shouldn't parent be `mContextMenu` to properly delete old actions on each right click?

Reply:
Thanks, fixed!